x-pack/plugins/screenshotting/server/screenshots/get_pdf.ts

```diff
@@ -27,6 +27,7 @@ export async function getPdf(
       data: await browser.printA4Pdf({ title, ...options }),
       title: null,
       description: null,
+      versionInfo: await browser.getVersion(),
```

Comment:
We really don't need there, since the version info is only needed for the tests. This PR tests the version string by getting a screenshot and using the `Screenshot` interface. Taking a screenshot is the easiest way to start interacting with the browser driver, which has the `getVersion()` method and we don't have access to that elsewhere. Unfortunately, this affects everything that returns a screenshot.